### PR TITLE
Readme copy

### PR DESCRIPTION
**Pull request by digitalbuddha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,19 @@
 
 ![build](https://github.com/twitter/compose-rules/actions/workflows/build.yaml/badge.svg) ![Latest version](https://img.shields.io/maven-central/v/com.twitter.compose.rules/common)
 
-A big challenge to face when a big team with a large codebase starts adopting Compose is that not everybody will start at the same page. This happened to us at Twitter.
+It can be challenging for bit teams to start adopting Compose, particularly because not everyone will start at same time or with same patterns. Twitter tried to ease the pain by creating a set of Compose Lint rules.
```
Comment (kenyee) [inline README.md:6]: typo:
"bit" = "big"

Comment (kenyee) [reply README.md:6]: typo "at same time" should be "at the same time"

"with same patterns" should be "with the same patterns"

File: README.md [hunk 2]
```diff
@@ -3,17 +3,19 @@
...
 
-Compose is 🔝, allows for amazing things, but has a bunch of footguns to be aware of. [See the thread](https://twitter.com/mrmans0n/status/1507390768796909571).
+Compose has lots of superpowers but also has a bunch of footguns to be aware of. [as seen in this Twitter Thread](https://twitter.com/mrmans0n/status/1507390768796909571).
```
Comment (kenyee) [inline README.md:8]: "." should be after the link the the thread.

File: README.md [hunk 3]
```diff
@@ -3,17 +3,19 @@
...
 
-This is where these static checks come in. We want to detect all the potential issues even before they reach the code review state, and foster a healthy Compose adoption.
+This is where our static checks come in. We want to detect as many potential issues as we can, as quickly as we can. In this case we want to fail lint prior to engineers having to review code, similar to other static check libraries we hope this leads to a "don't shoot the messengers" philosphy which will foster healthy Compose adoption.
```
Comment (kenyee) [inline README.md:10]: "Similar to other static check libraries..." should be a new sentence

File: README.md [hunk 4]
```diff
@@ -3,17 +3,19 @@
...
 
 Check out the project website for more information: https://twitter.github.io/compose-rules
 
 ## Static checks for Compose bundled in this repo
 
-The comprehensive list of what these rules will check for is in [the rules documentaton](https://github.com/twitter/compose-rules/blob/main/docs/rules.md). It contains both what we check for and why are we doing that, so giving it a good read is encouraged.
+You can find the comprehensive list of rules will check for in [the rules documentaton](https://twitter.github.io/compose-rules/rules). It contains both the what and why for each rule, we encourage you to read it prior to adopting our rules.
```
Comment (kenyee) [inline README.md:16]: "list of rules will check for" should be "list of rules"

**Comment by digitalbuddha** [comment]
recheck

File: README.md (hunk 2 above)
Comment (mrmans0n) [inline README.md:8]: That rogue dot probably has to go "...to be aware of[.] as seen in this..."

File: README.md [hunk 5]
```diff
@@ -41,7 +43,7 @@ For more information, you can refer to the [Using with Detekt](https://twitter.g
 
 ## Contributing
 
-Contributing new rules or fixes is welcome. See the [Contributing](CONTRIBUTING.md) instructions for more information.
+We love sharing and learning from others, contributing new rules or fixes is welcome. See the [Contributing](CONTRIBUTING.md) instructions for more information.
```
Comment (kenyee) [inline README.md:46]: should replace "," with ";"

**Review by kenyee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (digitalbuddha) [reply README.md]: just caught the bit one, on the other one, btw you can do it as a suggestion so I can auto apply

**Review by mrmans0n — APPROVED** [review]
Remove the dot and good to go

File: README.md (hunk 2 above)
Comment (kenyee) [reply README.md:8]: " aware of. [as " should be " aware of, [as "

**Comment by digitalbuddha** [comment]
I think that's all the comments, I can't merge until approved or maybe I'm not fancy enough to merge at all

**Comment by mrmans0n** [comment]
You'll need to say the "I have read the CLA Document and I hereby sign the CLA" btw, only signing the CLA without that didn't work for me (I tried rechecking and still fails your CLA compliance)

**Comment by digitalbuddha** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by digitalbuddha** [comment]
hmm still nothing, going to go for a walk and try again  and go to sleep Nacho!

**Comment by mrmans0n** [comment]
I've forced the recheck and it should pass now.

**Review by kenyee — APPROVED** [review]

**Pull request merged**